changelogs/fragments/4520-xfconf-deprecate-disable-facts.yml

```diff
@@ -0,0 +1,2 @@
+deprecated_features:
+  - xfconf - Deprecated parameter ``disable_facts``, as since version 4.0.0 it only allows value ``true`` (https://github.com/ansible-collections/community.general/pull/4520).
```

Comment:
```suggestion
  - xfconf - deprecated parameter ``disable_facts``, as since version 4.0.0 it only allows value ``true`` (https://github.com/ansible-collections/community.general/pull/4520).
```